composer/loggers/mlflow_logger.py

```diff
@@ -150,6 +152,23 @@ def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
             )
             self._optimized_mlflow_client.flush(synchronous=False)
 
+    def log_model(self, flavor: str, **kwargs):
+        """Log a model to MLFlow.
+
+        Args:
+            flavor (str): The MLFlow model flavor to use. Currently only ``'transformers'`` is supported.
+            **kwargs: Keyword arguments to pass to the MLFlow model logging function.
+
+        Raises:
+            NotImplementedError: If ``flavor`` is not ``'transformers'``.
+        """
+        if self._enabled:
+            import mlflow
+            if flavor == 'transformers':
+                mlflow.transformers.log_model(**kwargs,)
```

Comment:
i dont think we have a better approach, but worth noting this call will not be handled via the batching/async logging mechanism used in `log_metric`/`log_param`.

Reply:
Yeah, will it upload the multiple files associated with a huggingface checkpoint concurrently? or is it really just sending the whole object up in one chunk?

Reply:
We have multipart upload logic so I think it should be fairly performant

Reply:
great